.github/workflows/ruby.yml

```
@@ -0,0 +1,58 @@
+name: Ruby
```

```suggestion
name: CI
```
To get the `CI` on the badge